lms/urls.py

```diff
@@ -91,6 +91,8 @@
     # independently of courseware. https://github.com/edx/edx-val
     url(r'^api/val/v0/', include('edxval.urls')),
 
+    # Full Course/Library Import/Export API
+    url(r'^api/import_export/v1/', include('openedx.core.djangoapps.import_export.urls')),
```

Comment:
My big concern with it is just that the modulestore as configured for the LMS doesn't currently allow you to write to content or settings scopes.